packages/firebase_messaging/firebase_messaging/ios/firebase_messaging/Sources/firebase_messaging/FLTFirebaseMessagingPlugin.m

```diff
@@ -370,8 +370,11 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
   // coming from a terminated state.
   if (_notificationOpenedAppID != nil &&
       ![_initialNotificationID isEqualToString:_notificationOpenedAppID]) {
-    NSDictionary *notificationDict =
-        [FLTFirebaseMessagingPlugin remoteMessageUserInfoToDict:remoteNotification];
+    NSMutableDictionary *notificationDict =
+        [[FLTFirebaseMessagingPlugin remoteMessageUserInfoToDict:remoteNotification] mutableCopy];
+    if (response.actionIdentifier != nil) {
+      notificationDict[@"actionIdentifier"] = response.actionIdentifier;
```

Comment:
Leave a note that this is a property on RemoteMessage like:

``` 
// message.actionIdentifier
```
Example: https://github.com/Hackmodford/flutterfire/blob/feat(firebase_messaging)--Add-ios-action-identifier/packages/firebase_messaging/firebase_messaging/ios/firebase_messaging/Sources/firebase_messaging/FLTFirebaseMessagingPlugin.m#L848